content/docs/addons-test-utils.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: test-utils
-title: Test Utilities
+title: Utilitas Tes
 permalink: docs/test-utils.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**Cara Import**
 
 ```javascript
 import ReactTestUtils from 'react-dom/test-utils'; // ES6
-var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
+var ReactTestUtils = require('react-dom/test-utils'); // ES5 dengan npm
 ```
 
-## Overview {#overview}
+## Ikhtisar {#ikhtisar}
 
-`ReactTestUtils` makes it easy to test React components in the testing framework of your choice. At Facebook we use [Jest](https://facebook.github.io/jest/) for painless JavaScript testing. Learn how to get started with Jest through the Jest website's [React Tutorial](http://facebook.github.io/jest/docs/en/tutorial-react.html#content).
+`ReactTestUtils` mempermudah kita melakukan tes pada komponen React dengan _framework_ tes pilihan Anda. Di Facebook kami menggunakan [Jest](https://facebook.github.io/jest/) untuk tes JavaScript yang tidak merepotkan. Belajar cara mulai menggunakan Jest melalui situs Jest [React Tutorial](http://facebook.github.io/jest/docs/en/tutorial-react.html#content).
 
-> Note:
+> Catatan:
 >
-> We recommend using [`react-testing-library`](https://git.io/react-testing-library) which is designed to enable and encourage writing tests that use your components as the end users do.
+> Kami menyarankan Anda untuk menggunakan [`react-testing-library`](https://git.io/react-testing-library) yang didesain untuk memungkinkan dan mendorong penulisan tes yang menggunakan komponen seperti pengguna.
```

Comment:
`... yang didesain untuk memungkinkan dan mendorong penulisan tes yang menggunakan komponen seperti pengguna.`

`... yang didesain untuk memungkinkan dan mendorong penulisan tes yang akan menggunakan komponen anda selayaknya seorang pengguna sebenarnya`

What do you think?

Reply:
for me i think the first one is clear enough, but the second one also fine so need other's opinion on this @regalius

Reply:
I think "that use your components as the end users do" refers to `react-testing-library` as in "`react-testing-library` uses your components as the end users do to write tests". Thoughts?

My input would be:
`... yang didesain untuk memfasilitasi dan mendorong penulisan tes dengan menggunakan komponen Anda sebagaimana (selayaknya) seorang pengguna sebenarnya.`

Reply:
@spondbob good input on `memfasilitasi` haha will change it